<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QMainWindow" name="LogIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(1, 116, 190);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_Log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1201</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/image/qt_log_p.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_safe_sing">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>661</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QFrame" name="frame_safe_2l">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/image/rang1.png);
background-color: rgb(12, 53, 106);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_safe_3l">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/image/rang1.png);
background-color: rgb(12, 53, 106);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_safe_4l">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/image/rang1.png);
background-color: rgb(12, 53, 106);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_show_safe_l">
      <property name="geometry">
       <rect>
        <x>452</x>
        <y>30</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
image: url(:/new/prefix1/image/rang1.png);
background-color: rgb(12, 53, 106);</string>
      </property>
      <property name="text">
       <string>نمایش عدد</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_safe_1l">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/image/rang1.png);
background-color: rgb(12, 53, 106);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_log_name">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>70</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_log_Password">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_log_safe">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <zorder>lineEdit_log_name</zorder>
    <zorder>lineEdit_log_Password</zorder>
    <zorder>lineEdit_log_safe</zorder>
    <zorder>groupBox_safe_sing</zorder>
   </widget>
   <widget class="QFrame" name="frame_Poetry_log">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>951</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/image/qt_Poetry.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ok_log">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12, 53, 106);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>تایید</string>
    </property>
   </widget>
   <zorder>frame_Poetry_log</zorder>
   <zorder>frame_Log</zorder>
   <zorder>pushButton_ok_log</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
